lib/cocoapods/downloader/cache.rb

```diff
@@ -304,55 +279,23 @@ def in_tmpdir(&blk)
       #
       # @return [Void]
       #
-      def copy_source_and_clean(source, destination, spec)
-        specs_by_platform = group_subspecs_by_platform([spec])
+      def copy_and_clean(source, destination, spec)
+        specs_by_platform = group_subspecs_by_platform(spec)
         destination.parent.mkpath
         Cache.write_lock(destination) do
-          Pod::Executable.execute_command('rsync', ['-a', '--exclude=.git/fsmonitor--daemon.ipc', '--delete', "#{source}/", destination])
+          FileUtils.rm_rf(destination)
+          FileUtils.cp_r(source, destination)
```

Comment:
Looks like the issue is that when this line copies a link and the link points to already existing file then it trow the error. cc @dnkoutso